src/creature/creature.cpp

```diff
@@ -39,4 +40,17 @@ void Creature::PrintStats()
 	std::cout << "==============================" << std::endl;
 }
 
-int Creature::get_hp() { return m_hp; }
+int Creature::get_hp() { return m_hp; };
+int Creature::get_ability_score(int n)
+{
+	if (n >= -1 && n < 6)
+	{
+		return m_ability_scores[n];
+	}
+	else
+	{
+		// Return a default value for invalid ability scores
```

Comment:
I like the negative return value here. I would also have the following statement before the return:
`std::cerr << "You have asked for an invalid ability score. Valid values are between 0 and 6." << std::endl;`